include/psa/crypto_types.h

```diff
@@ -226,15 +233,84 @@ typedef uint32_t psa_key_location_t;
  * - 0 is reserved as an invalid key identifier.
  * - Key identifiers outside these ranges are reserved for future use.
  */
-/* Implementation-specific quirk: The Mbed Crypto library can be built as
- * part of a multi-client service that exposes the PSA Crypto API in each
- * client and encodes the client identity in the key id argument of functions
- * such as psa_open_key(). In this build configuration, we define
- * psa_key_id_t in crypto_platform.h instead of here. */
-#if !defined(MBEDTLS_PSA_CRYPTO_KEY_FILE_ID_ENCODES_OWNER)
 typedef uint32_t psa_key_id_t;
-#define PSA_KEY_ID_INIT 0
-#endif
+
+#if !defined(MBEDTLS_PSA_CRYPTO_KEY_ID_ENCODES_OWNER)
+typedef psa_key_id_t mbedtls_svc_key_id_t;
+
+#define MBEDTLS_SVC_KEY_ID_INIT ( (psa_key_id_t)0 )
```

Comment:
Not very important since the distinction between `crypto_types.h` and `crypto_values.h` is purely cosmetic, but in principle, macros and functions to analyze or construct values should be in `crypto_values.h`, not `crypto_types.h`.

Reply:
Better to put away things as they should, thus kept in crypto_types.h just the type definitions and moved the rest to crypto_values.h.